### PR TITLE
Update readme files regarding reference on global plugin parameters

### PR DESCRIPTION
**Pull request by Softwarecules**
## Summary

Updated the plugin.fields part of the @polar/core documentation to make it easier to understand and navigate. Also added a reference to the global parameters part in every plugin documentation in the configuration section.

## Instructions for local reproduction and review
Check the `README.md` files.

### CODE REVIEW

**Comment by dopenguin** [comment]
I've also realized that the links won't be working on the online documentation for the clients (e.g https://dataport.github.io/polar/docs/afm/plugin-draw.html)

My guess would be that the file `scripts/makeDocs.js` needs an adjustment that replaces the relative path with a link to the correct place of the documentation relevant for the client

File: packages/plugins/Export/README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ The Export plugin offers users to download the currently visible map canvas in a
 | showPdf | boolean? | Tools offers current map view as PDF. Defaults to `true`. |
 | showPng | boolean? | Tools offers current map view as PNG. Defaults to `true`. |
 
+For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
+
 ## Store
 
 To remove user control, add `displayComponent: false` to the configuration and use the following interactions.
```
[the remark this answers is not in the document]
Comment (Softwarecules) [reply packages/plugins/Export/README.md:22]: Hmm I don't understand this one 🤔 Above the new line? What do you mean?

[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/plugins/Export/README.md:22]: The sentence `To remove user control, add `displayComponent: false` to the configuration and use the following interactions.` should be adjusted and placed above the new sentence `For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.`

File: packages/plugins/ReverseGeocoder/README.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ This module has been written for the HH WPS service. The return format is custom
 | removeLoading | string? | Points to an action in the store; commited with a loading key as payload on finishing reverse geocoding. |
 | zoomTo | number? | If specified, plugin zooms to given coordinate after successful reverse geocoding; number indicates maximal zoom level. |
 
+For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
+
```
[the remark this answers is not in the document]
Comment (Softwarecules) [reply packages/plugins/ReverseGeocoder/README.md]: FIxed 👍

File: packages/plugins/Export/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Softwarecules) [reply packages/plugins/Export/README.md:18]: FIxed 👍 ``

File: packages/plugins/Pins/README.md [hunk 3]
```diff
@@ -22,6 +22,8 @@ The usage of `displayComponent` has no influence on the creation of Pins on the
 | toastAction | string? | If `boundaryLayerId` is set, and the pin is moved or created outside the boundary, this string will be used as action to send a toast information to the user. If no toast information is desired, leave this field undefined; for testing purposes, you can still find information in the console. |
 | toZoomLevel | number? | Zoom level to use on outside input by e.g. address search. Defaults to `0`. |
 
+For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
+
```
[the remark this answers is not in the document]
Comment (Softwarecules) [reply packages/plugins/Pins/README.md]: FIxed 👍

File: packages/clients/meldemichel/CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 ## unpublished
 
 - Chore: Change value of `pins.movable` configuration to `'drag'` as using a boolean has been deprecated in a future release.
+- Chore: Removed the `displayComponent` attribute for `@polar/plugin-pins` as it is not necessary.
```
Comment (dopenguin) [inline packages/clients/meldemichel/CHANGELOG.md:6]:
```suggestion
```
Not relevant for the consumer of this client

**Review by dopenguin — CHANGES_REQUESTED** [review]
Merging the `defaultOptions` in `@polar/client-snowbox` is not relevant as well.

Looks good otherwise 🏓 @Softwarecules

File: packages/core/README.md [hunk 5]
```diff
@@ -74,7 +74,7 @@ The mapConfiguration allows controlling many client instance details.
 | layerConf | LayerConf | Layer configuration as required by masterportalAPI. |
 | language | enum["de", "en"] | Initial language. |
 | <...masterportalAPI.fields> | various | The object is also used to initialize the masterportalAPI. Please refer to their documentation for options. |
-| <plugin.fields> | various? | Many plugins added with `addPlugin` may respect additional configuration. Please see the respective plugin documentations. Global plugin parameters are described below. |
+| <plugin.fields> | various? | Fields for configuring plugins added with `addPlugins`. Refer to each plugin's documentation for specific fields and options. Global plugin parameters are described [below](../core/README.md#global-plugin-parameters). |
```
Comment (dopenguin) [inline packages/core/README.md:77]:
```suggestion
| <plugin.fields> | various? | Fields for configuring plugins added with `addPlugins`. Refer to each plugin's documentation for specific fields and options. Global plugin parameters are described [below](#global-plugin-parameters). |
```

File: packages/core/README.md [hunk 6]
```diff
@@ -199,21 +199,32 @@ The `<...masterportalAPI.fields>` means that any masterportalAPI field may also
 
 ##### <plugin.fields>
 
-On how to configure a plugin, see the respective plugin. The configuration is given in the `mapConfiguration` object by the plugin's name as specified in its respective documentation. For example, a `@polar/plugin-address-search` plugin can be configured like this:
+Plugins in POLAR are modular components that extend the functionality of the map client. They can be added using the [addPlugins](../core/README.md#addPlugins) method and configured through the `mapConfiguration` object. Each plugin has its own set of fields and options that can be customized.
```
Comment (dopenguin) [inline packages/core/README.md:202]:
```suggestion
Plugins in POLAR are modular components that extend the functionality of the map client. They can be added using the [addPlugins](#addPlugins) method and configured through the `mapConfiguration` object. Each plugin has its own set of fields and options that can be customized.
```

**Review by dopenguin — CHANGES_REQUESTED** [review]
> > Merging the `defaultOptions` in `@polar/client-snowbox` is not relevant as well.
> > Looks good otherwise 🏓 @Softwarecules
> 
> I don't really understand what you mean by this^^ I didn't touch the `defaultOptions`? Otherwise I am done now @dopenguin 🏓

The `defaultOptions` are merged in for `Pins` in `@polar/client-snowbox`. This is not be needed and should be changed while we are at it.

I've added two suggestions which for the MarkDown file but not for the resulting html (the headings are missing ids, see [here](https://developers.google.com/style/headings-targets)).

🏓 @Softwarecules

**Review by dopenguin — APPROVED** [review]
🚀 @Softwarecules

**Pull request merged**